playwright/core/expectScreenshotFixture.ts

```diff
@@ -10,6 +10,7 @@ export const expectScreenshotFixture: PlaywrightFixture<ExpectScreenshotFixture>
     const expectScreenshot: ExpectScreenshotFixture = async ({
         component,
         screenshotName,
```

Comment:
Let's remove `screenshotName` (it's not used anywhere), and instead of `screenshotPostfix` introduce `nameSuffix` option

Reply:
Done

![image](https://github.com/user-attachments/assets/94309773-c12d-4f97-9571-fd64df7324d5)